app/data.py

```diff
@@ -57,3 +57,10 @@ def drop_index(self, collection: str):
 
     def search(self, collection: str, user_search: str) -> List[Dict]:
         return self.read(collection, {"$text": {"$search": user_search}})
+
+    def scan_collections(self):
+        output = {}
+        cli = MongoClient(getenv("MONGO_URL"))[self.cluster]
+        for col in cli.list_collection_names():
+            output[col] = self.count(col, {})
```

Comment:
This could be a dictionary comprehension.

Reply:
`return {col: self.count(col, {}) for col in cli.list_collection_names()}`

Reply:
Dictionary comprehension isn't something that jumps into my mind yet, I need to spend more time using it so that I can see where to use it more